crypto/rsa_extra/rsa_asn1.c

```diff
@@ -188,7 +214,12 @@ RSA *RSA_parse_private_key(CBS *cbs) {
     goto err;
   }
 
-  if (!RSA_check_key(ret)) {
+  rsa_asn1_key_encoding_t rsa_enc_key_type = RSA_CRT_KEY;
+  if (detect_stripped_accp_private_key(ret) == 1) {
```

Comment:
```suggestion
  if (detect_stripped_jca_private_key(ret) == 1) {
```

Reply:
i have no issue with that. perhaps we can consolidate discussion with @SalusaSecondus around the naming issue on [this thread](https://github.com/awslabs/aws-lc/pull/436#discussion_r850827317)?